### ISSUE TITLE
problems with non-English letters when server topic is non-english

### ISSUE THREAD

**Issue by jonjamcam** [elieserdejesus/JamTaba#498]
I noticed this happens with my Spanish server Topic contrary to reaninjam's behavior that recognizes accents correctly:

![image](https://cloud.githubusercontent.com/assets/15310433/16890110/b507c924-4ab9-11e6-9305-f8e44710caf7.png)

**Comment by elieserdejesus** [comment]
@jonjamcam , I confirm the problem here with my local server. 
I do some experiments to try understand the problem and my conclusion is: the **.cfg** file used when open the local server is not encoded in utf-8, I confirmed this opening the cfg file in a utf-8 capable text editor (and I see wrong characters). 

Jamtaba is assumming utf-8 in every strings, but this string (the topic text) readed from a external file is an exception. This exception is easy to solve, it's just detect when we are receiving a topic message and use a non utf-8 decoder. I tested here and works.

I see 2 problems:
1 - if we change the topic to non utf-8 people can't write japanese (or other non ascii) texts for topic.
2 - I'm not shure if ninbot.com and ninjamer.com (the most used public servers) are using the official server. Maybe Tom and Damned changed the server to read the config file using utf-8.

I'm not shure how to solve this at moment.

**Comment by jonjamcam** [comment]
> I'm not shure if ninbot.com and ninjamer.com (the most used public servers) are using the official server

ninbot.com topic works with japanese letters (ninbot.com allows change user's topic using the **nb _whatever topic_** command):

![image](https://cloud.githubusercontent.com/assets/15310433/16903240/cabd6e84-4c46-11e6-90d8-50eba69e5817.png)

so probably it's not a Jamtaba problem. 

> the .cfg file used when open the local server is not encoded in utf-8, I confirmed this opening the cfg file in a utf-8 capable text editor (and I see wrong characters).

I saved my own config file using utf-8 and now there's no problem.

![image](https://cloud.githubusercontent.com/assets/15310433/16903271/c9de31d2-4c47-11e6-8811-10365b55848f.png)

So I think there's nothing to solve. Server's admins should save always using utf-8 encoding to avoid this behavior. Closing.....
